truss/templates/server/truss_server.py

```diff
@@ -66,33 +66,25 @@ def __init__(self, model: ModelWrapper, tracer: sdk_trace.Tracer) -> None:
         self._model = model
         self._tracer = tracer
 
-    def _safe_lookup_model(self, model_name: str = MODEL_BASENAME) -> ModelWrapper:
-        if model_name != self._model.name:
-            raise errors.ModelMissingError(model_name)
-        return self._model
-
-    @staticmethod
-    def check_healthy(model: ModelWrapper):
```

Comment:
Didn't make sense for this to be static, so we no longer need to pass the `model` explicitly